packages/models/src/classMap.js

```diff
@@ -176,7 +176,6 @@ export default class ClassMap {
   }
 
   toJSON() {
-    // Don't write out code objects that were created during runtime
-    return this.roots.filter((obj) => !obj.dynamic);
```

Comment:
It's not entirely clear to me why this was removed.

Reply:
This filter was never working anyway because the it's actually `obj.data.dynamic`. In any case, when we write the `classMap.json`, we want to include all code objects, whether generated or not.